.coveragerc

```diff
@@ -709,6 +709,9 @@ omit =
     homeassistant/components/nfandroidtv/notify.py
     homeassistant/components/niko_home_control/light.py
     homeassistant/components/nilu/air_quality.py
+    homeassistant/components/nina/__init__.py
+    homeassistant/components/nina/binary_sensor.py
+    homeassistant/components/nina/const.py
```

Comment:
It is not compulsory, but since new integrations can take a while to get merged, I suggest that you take the opportunity to move to 100% coverage whilst the reviews are in progress.